src/main/java/org/jenkinsci/plugins/docker/workflow/WithContainerStep.java

```diff
@@ -186,7 +187,7 @@ private static void destroy(String container, Launcher launcher, Node node, EnvV
                         throw new IOException("failed to run ps");
                     }
                     List<String> pids = new ArrayList<String>();
-                    LINE: for (String line : baos.toString().split("\n")) {
+                    LINE: for (String line : baos.toString(Charset.defaultCharset().name()).split("\n")) {
                         for (Map.Entry<String,String> entry : modelEnvVars.entrySet()) {
```

Comment:
Well in those other places the value was used more than once. Here it is only calculated on time, so I don't see a need for an extra variable, except for maybe readability.